src/lib/components/Identifiers/PIDField.js

```diff
@@ -55,12 +55,17 @@ class UnreservePIDBtn extends Component {
   render() {
     const { disabled, handleDiscardPID, loading } = this.props;
     return (
-      <Form.Button
-        disabled={disabled || loading}
-        loading={loading}
-        icon="close"
-        onClick={handleDiscardPID}
-        size="mini"
+      <Popup
+        content="Discard the reserved DOI"
```

Comment:
Please do not hardcode DOI :) It is a generic component that can handle any `PID`:
```suggestion
        content="Discard the reserved {pid_type}"
```